### PR TITLE
Search page

### PR DESCRIPTION
**Pull request by jeinygroove**
You can click on the `SEARCH` button now, input fields and push `Submit`, after that you will be redirected to the song page (song page still has only static data, you input won't affect it)

### CODE REVIEW

**Review by sashapff — APPROVED** [review]

**Comment by jeinygroove** [comment]
I just noticed that I allow user to left fields blank, not good

File: src/main/java/com/google/alpollo/HelloServlet.java [hunk 1]
```diff
@@ -1,16 +0,0 @@
...
-import java.io.IOException;
-
-@WebServlet("/hello")
-public class HelloServlet extends HttpServlet {
-  @Override
-  protected void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Alpollo says Hi! to the world!");
-  }
-}
```
Comment (jgaillard85) [inline src/main/java/com/google/alpollo/HelloServlet.java]: This has been done in another PR already

Comment (jeinygroove) [reply src/main/java/com/google/alpollo/HelloServlet.java]: fixed

**Review by amaurymedeiros — APPROVED** [review]
